ui/lib/core/package.json

```diff
@@ -14,6 +14,7 @@
     "ember-cli-clipboard": "*",
     "ember-cli-htmlbars": "*",
     "ember-cli-string-helpers": "*",
+    "ember-cli-typescript": "*",
```

Comment:
[Required for deps in addons](https://docs.ember-cli-typescript.com/ts/with-addons)